bokehjs/test/models/ranges/data_range1d.coffee

```diff
@@ -67,6 +67,12 @@ describe "datarange1d module", ->
     it "should have max = 20", ->
       expect(r.max).to.be.equal 20
 
+ describe "explicit inverted bounds=(20,10) creation", ->
+   r = new DataRange1d({start: 20, end:10})
+
+   it "should be reversed", ->
+     expect(r.is_reversed).to.be.equal true
+
```

Comment:
Just noting here that ideally we'd also have a test that initializes with `flipped=True` and lets `start` and `end` be filled in by auto-ranging, but that would be a pain to do unfortunately